### PR TITLE
First raw PR

### PR DESCRIPTION
**Pull request by cableman**
#### Link to ticket

https://jira.itkdev.dk/browse/KLIMA-15

#### Description

Start op with simple API platform for the entities and easy admin for editing data.

#### Screenshot of the result

N/A

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

N/A

### CODE REVIEW

File: src/Controller/Admin/AdminDashboardController.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+{
+    public function __construct(
+        private readonly AdminUrlGenerator $adminUrlGenerator
+    ) {
+    }
+
+    #[Route('/admin', name: 'admin')]
+    public function index(): Response
+    {
+        $d = $this->adminUrlGenerator
```
Comment (tuj) [inline src/Controller/Admin/AdminDashboardController.php:27]: $d ?

File: src/MessageHandler/WebhookHandler.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<?php
+
+// api/src/Handler/PersonHandler.php
```
Comment (tuj) [inline src/MessageHandler/WebhookHandler.php:3]: remove

File: src/MessageHandler/WebhookHandler.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        $tile = new Tile();
+
+        $this->em->persist($tile);
+        $this->em->flush();
+
+        $this->metricsService->counter('webhook_success_total', 'Webhook completed successful', 1, ['type' => 'webhook']);
+    }
+}
+
+// throw new ReQueueMessageException('Cover store error - retry');
```
Comment (tuj) [inline src/MessageHandler/WebhookHandler.php:41]: Remove

**Review by tuj — APPROVED** [review]
Approved with few comments

File: src/Controller/Admin/AdminDashboardController.php (hunk 1 above)
Comment (cableman) [reply src/Controller/Admin/AdminDashboardController.php:27]: c&p from easyAdmin getting started guide :-)

**Pull request closed without merging**